confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -311,7 +312,7 @@ def get_compatibility(self, subject=None):
         if code >= 200 and code <= 299:
             compatibility = result.get('compatibility', None)
```

Comment:
Looking at the REST API spec it seems that this value is never returned but `is_compatible`:
https://docs.confluent.io/current/schema-registry/docs/api.html#id1

There is also no mention of `compatibilityLevel` being returned.

So either the documentation is wrong or this function must never have worked.

Reply:
The REST API was updated. It did return `compatibility` in version `3.0` - https://docs.confluent.io/3.0.1/schema-registry/docs/api.html#config
And yeah, `compatibilityLevel` is not mentioned anywehere, even in version `2.0`

Reply:
Wow haha, nope. We're looking at the wrong API endpoint.
This calls GET /config
https://docs.confluent.io/current/schema-registry/docs/api.html#get--config
Which does return `compatibility`. Still no mention of `compatibilityLevel` though

Reply:
The question then becomes - what should we return when the response is a non  2xx status code - 'NONE' ?

Reply:
This method calls '/config':
https://github.com/confluentinc/confluent-kafka-python/blob/66a181459731515dc0bb86a9de69dcc51278797c/confluent_kafka/avro/cached_schema_registry_client.py#L300
https://github.com/confluentinc/confluent-kafka-python/blob/master/confluent_kafka/avro/cached_schema_registry_client.py#L306